wlutil/init.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from .wlutil import *
+from .br import br
+
+# Initialize wlutil for the first time
+def oneTimeInit():
+    log = logging.getLogger()
+
+    # Build initramfs fs structure (git can't save these because reasons)
+    initramfs_dirs = ["bin", 'dev', 'etc', 'proc', 'root', 'sbin', 'sys', 'usr/bin', 'usr/sbin', 'mnt/root']
+    for d in initramfs_dirs:
+        if not (initramfs_root / d).exists():
+            (initramfs_root / d).mkdir(parents=True)
+
+    # Make busybox (needed for the initramfs)
+    log.info("Building busybox (used in initramfs)")
+    shutil.copy(wlutil_dir / 'busybox-config', busybox_dir / '.config')
+    run(['make', jlevel], cwd=busybox_dir)
+    shutil.copy(busybox_dir / 'busybox', initramfs_root / 'bin/')
+
+    # Apply linux patches to the default kernel
+    patches = list(board_dir.glob("*.patch"))
```

Comment:
This step will go away once we get the patches upstreamed. I think it's a fair trade for not dealing with a custom fork. One downside will be that anyone using a custom kernel will need to address this themselves since we only patch the default kernel here.

Reply:
This has been removed in favor of a Linux fork that differs from upstream only in these two patches. We will strive to get off that and minimize changes.